src/main/java/de/doubleslash/keeptime/REST_API/mapper/ColorMapper.java

```diff
@@ -0,0 +1,30 @@
+package de.doubleslash.keeptime.REST_API.mapper;
```

Comment:
for new classes we need to add the license header (with current year)

Reply:
Done.